Add alerts config for fire/smoke

StefanSaramet opened the pull request:
* ignore mongo db folder
* add global usings and remove assemblies included in the Alerts svc
* add appsettings configs for fire/smoke

CESARDELATORRE commented:
Will the service still work with the original model with the "person" class if we merge this PR in MAIN?

StefanSaramet commented:
> Will the service still work with the original model with the "person" class if we merge this PR in MAIN?

It should be working just fine, I just added a new configuration for another detection that this service will receive. But I added locally an integration test for this controller in which I sent 2 payloads - a `person` and a `fire`. The Alert service processed both received detections successfully.

CESARDELATORRE commented:
We have the JSON code for the class "smoke" and related RULE "SmokeDetected" already in MAIN, so it's better to close this PR that has conflicts and if you'd like to request another PR for pure C# code refactoring, it's better to do it in a different PR specific for refactoring. Although that PR (just removing unused "usings") is not critical/needed.
